spec/cypress/support/axe.js

```diff
@@ -0,0 +1,16 @@
+import "cypress-axe";
+
+Cypress.Commands.overwrite("injectAxe", (orig, options) => {
+  orig(options);
+
+  cy.configureAxe({
+    rules: [{ id: "region", enabled: false }],
+  });
+});
+
+Cypress.Commands.overwrite("checkA11y", (orig, ...args) => {
+  // It doesn't seem to affect anything if injectAxe is ran multiple times
```

Comment:
I hope this comment won't come back to bite us, but oh well :D

Reply:
🤷‍♂️ it should fail obviously if it changes